app/static/stock/stock_detail.js

```diff
@@ -72,20 +73,21 @@ $(window).resize(function () {
 });
 
 
-function add_or_remove_stock(checkbox, symbol) {
-    let add = null
-    if (checkbox.checked) {
-        add = 1
-    } else {
-        add = 0
-    }
-
+function add_or_remove_stock(checkbox, symbol) {    
+    let add = checkbox.checked;
     $.ajax({
         data: {symbol: symbol, append: add}, 
         type: 'POST',
         url: '/stock/add_to_watch_list'
     }).done(function(reponse) {
-
+        let process = 'removed from';
+        if (checkbox.checked) {
```

Comment:
Should this be `if (add) {`?

Reply:
add is checkbox.checked. I thought it was weird to use add.

Reply:
Sounds good.